File: Curse/src/main/java/Model/Inscriere.java
package Model;

public class Inscriere extends Entity<Long> {
    public long Pilot;
    public long Cursa;

    public long getPilot() {
        return Pilot;
    }

    public void setPilot(long pilot) {
        Pilot = pilot;
    }

    public long getCursa() {
        return Cursa;
    }

    public void setCursa(long cursa) {
        Cursa = cursa;
    }

    public Inscriere(long pilot, long cursa) {
        Pilot = pilot;
        Cursa = cursa;
    }
}

File: CurseCS/Model/Inscriere.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace curse.Model
{
    [Serializable]
    public class Inscriere : Entity<long>
    {
        public long Pilot { get;  set; }
        public long Cursa { get;  set; }
        public Inscriere(long pilot, long cursa)
        {
            Id = 0;
            Pilot = pilot;
            Cursa = cursa;
        }
        public Inscriere(long id, long pilot, long cursa)
        {
            Id = id;
            Pilot = pilot;
            Cursa = cursa;
        }

        public long getPilot()
        {
            return Pilot;
        }
    }
}
